packages/mongodb-chatbot-server/src/test/testConfig.ts

```diff
@@ -47,7 +48,7 @@ const allowedOrigins = process.env.ALLOWED_ORIGINS?.split(",") || [];
   Boost results from the MongoDB manual so that 'k' results from the manual
   appear first if they exist and have a min score of 'minScore'.
  */
-export const boostManual = makeBoostOnAtlasSearchFilter({
+export const boostManual: SearchBooster = makeBoostOnAtlasSearchFilter({
```

Comment:
Why the explicit type annotation? Shouldn't `makeBoostOnAtlasSearchFilter` already resolve to a `SearchBooster` based on its signature?

Reply:
removing